### PR TITLE
feat: support reload from db when cluster config update

### PR DESCRIPTION
**Pull request by jamesgetx**
N

### CODE REVIEW

**Review by jamesgetx — COMMENTED** [review]
相关评论在 https://github.com/TencentBlueKing/blueking-paas/pull/1837/files

File: apiserver/paasng/paas_wl/infras/resources/base/base.py [hunk 1]
```diff
@@ -31,16 +34,29 @@
...
 
     NOTE: This function is cached for performance. When the clusters have been updated,
     the cache must be cleared.
+
+    :param version: The global config version. 与 lru_cache 配合使用, 如果 version 发生变化, 则缓存失效, 从数据库重新加载
     """
     return ContextConfigurationPoolMap.from_db()
 
 
+def invalidate_get_global_configuration_pool():
```
Comment (narasux) [inline apiserver/paasng/paas_wl/infras/resources/base/base.py:55]: Nit: invalidate_global_configuration_pool

File: apiserver/paasng/paas_wl/infras/resources/base/base.py [hunk 2]
```diff
@@ -112,3 +128,23 @@ def get_client_by_cluster_name(cluster_name: str) -> EnhancedApiClient:
...
+        if v := cache.get(self._key):
+            return v
+
+        v = timezone.now().strftime("%Y%m%d%H%M%S")
+        cache.set(self._key, v, timeout=None)
+        return v
+
+    def update(self):
+        """update last modified to indicate global config has been updated"""
+        v = timezone.now().strftime("%Y%m%d%H%M%S")
```
Comment (narasux) [inline apiserver/paasng/paas_wl/infras/resources/base/base.py:149]: 直接 timestamp ？

**Review by piglei — APPROVED** [review]

File: apiserver/paasng/paas_wl/infras/resources/base/base.py (hunk 2 above)
Comment (narasux) [reply apiserver/paasng/paas_wl/infras/resources/base/base.py:149]: int(timezone.now().timestamp())

Comment (narasux) [reply apiserver/paasng/paas_wl/infras/resources/base/base.py:149]: 或者 `%Y%m%d%H%M%S` 抽成类的常量

Comment (jamesgetx) [reply apiserver/paasng/paas_wl/infras/resources/base/base.py]: 调整了下

File: apiserver/paasng/paas_wl/infras/resources/base/base.py (hunk 1 above)
Comment (jamesgetx) [reply apiserver/paasng/paas_wl/infras/resources/base/base.py]: done

**Pull request merged**